iroha-cli/interactive/interactive_transaction_cli.hpp

```diff
@@ -117,6 +117,8 @@ namespace iroha_cli {
           std::vector<std::string> line);
       std::shared_ptr<iroha::model::Command> parseSetQuorum(
           std::vector<std::string> line);
+      std::shared_ptr<iroha::model::Command> parseSetAccountDetail(
+          std::vector<std::string> line);
```

Comment:
Ah, it seems all the other `parse*` functions use wrong signature :( Could you please change those as well? (related to previous comment)